charts/tidb-operator/templates/scheduler-deployment.yaml

```diff
@@ -94,4 +94,8 @@ spec:
       tolerations:
 {{ toYaml . | indent 8 }}
     {{- end }}
+    {{- with .Values.advancedStatefulset.securityContext }}
```

Comment:
why all of these templates refer to `securityContext` in `advancedStatefulset`?

Reply:
I'll fix